erpnext/setup/doctype/territory/territory.py

```diff
@@ -8,16 +8,21 @@
 from frappe.utils import flt
 from frappe import _
 
-from frappe.utils.nestedset import NestedSet
+from frappe.utils.nestedset import NestedSet, get_root_of
 
 class Territory(NestedSet):
 	nsm_parent_field = 'parent_territory'
 
 	def validate(self):
+		self.set_parent()
 		for d in self.get('targets') or []:
 			if not flt(d.target_qty) and not flt(d.target_amount):
 				frappe.throw(_("Either target qty or target amount is mandatory"))
 
+	def set_parent(self):
+		if not self.parent_territory:
```

Comment:
You cannot set parent territory forcefully, this way none will be able to create multiple root territories. You can set only default value.